### PR TITLE
feat(api): add policy exceptions service

### PR DESCRIPTION
**Pull request by dmurray-lacework**
Signed-off-by: Darren Murray <darren.murray@lacework.net>

## Summary

Add new policy exceptions service to interact with `v2/Exceptions` schema

## How did you test this change?

Manually run `examples/policy_exceptions/main.go`
Add unit tests

## Issue

https://lacework.atlassian.net/browse/ALLY-1046

### CODE REVIEW

**Review by afiune — COMMENTED** [review]
Code looks good, I just have one suggestion, wouldn't it make more sense
to nest this service inside the `Policy` service? This is just UX/aesthetics

Then we could do this instead `lacework.Policy.Exception.Create()`

File: api/policy_exceptions.go [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+
+type PolicyExceptionsResponse struct {
+	Data []PolicyException `json:"data"`
+}
+type PolicyException struct {
+	ExceptionID    string                      `json:"exceptionId,omitempty"`
+	Description    string                      `json:"description"`
+	Constraints    []PolicyExceptionConstraint `json:"constraints"`
+	LastUpdateTime string                      `json:"lastUpdateTime,omitempty"`
+	LastUpdateUser string                      `json:"lastUpdateUser,omitempty"`
```
Comment (afiune) [inline api/policy_exceptions.go:103]: can these fields be `time.Time`?

**Review by afiune — APPROVED** [review]
One comment but out of that, looks good.

File: api/policy_exceptions.go (hunk 1 above)
Comment (dmurray-lacework) [reply api/policy_exceptions.go:103]: LastUpdateTime, probably yeh. Though , from looking at our other lastupdatedtime fields we often use string. LastUpdateTime is only returned in the response, in terraform it'll be a computed value. A user will never have to construct a date for this field in a request. If we want to have this field also as a date type, I think having a method on PolicyException struct might be a good idea instead

**Pull request merged**